cmd/ferretdb/main.go

```diff
@@ -204,9 +214,86 @@ var (
 
 func main() {
 	setCLIPlugins()
-	kong.Parse(&cli, kongOptions...)
+	ctx := kong.Parse(&cli, kongOptions...)
+
+	switch ctx.Command() {
+	case "run":
+		run()
+	case "ping":
+		ping()
+	default:
+		panic("unknown sub-command")
+	}
+}
+
+// ping creates connection to FerretDB instance specified by the flags, and runs `ping` command against it.
+func ping() {
+	logger := setupLogger(setupState(), cli.Log.Format)
+	l := logger.Sugar()
+
+	checkFlags(logger)
```

Comment:
Do we really want this check? It means it will only work for new authentication.
https://github.com/FerretDB/FerretDB/blob/main/cmd/ferretdb/main.go#L302-L304

Reply:
Yes, it's expected for it to work only for a new authentication. Otherwise we don't have credentials to properly `ping` the backend.

Reply:
Ah I see. Let's leave that comment somewhere, possibly on the command?